### PR TITLE
Add syntactic support for tags 🏷

### PR DESCRIPTION
**Pull request by matheuss**
There's already technical support since you can `hyperpower#beta` in your `~/.hyperterm.js`, but syntactic support with proper documentation is better ☺️

### CODE REVIEW

**Comment by matheuss** [comment]
Just realized that I the code needs to be aware of `@org/project@beta` 😅 I'll add more commits ASAP

**Comment by matheuss** [comment]
Done ☺️ 

`.hyperterm.js`:
```js
plugins: [
    '@org1/project1@alpha',
    'project2',
    'project3@beta',
    '@org2/project4'
  ],
```
`.hyperterm_plugins/package.json`:
```js
"dependencies": {
    "org1/project1": "alpha",
    "project2": "latest",
    "project3": "beta",
    "@org2/project4": "latest"
  }
```

File: app/plugins.js [hunk 1]
```diff
@@ -190,8 +190,20 @@ function alert (message) {
...
+    tmp[pieces[0]] = null == pieces[1] ? 'latest' : pieces[1];
+    pieces = plugin.split('@');
+    if (pieces.length === 2) { // @org/project || project@tag
+      if (!plugin.startsWith('@')) {
+        tmp = {};
+        tmp[pieces[0]] = pieces[1];
+      }
+    } else if (pieces.length === 3) { // @org/project@tag
+      tmp = {};
+      tmp[`${pieces[1]}`] = pieces[2];
```
Comment (rauchg) [inline app/plugins.js]: why `?

Comment (matheuss) [inline app/plugins.js]: Ops, my bad 😅

**Comment by matheuss** [comment]
I'm closing this for now, I'll reopen later if necessary ^~^

**Pull request closed without merging**